addon/components/cf-field/input/table.js

```diff
@@ -97,6 +102,8 @@ export default Component.extend(ComponentQueryManager, {
 
         this.set("showModal", false);
       } catch (e) {
+        // eslint-disable-next-line no-console
+        console.error(e);
```

Comment:
Do we really need log the error when we already show a message?

Reply:
This is just supposed to help with analyzing why the generic error message shows up. Ideally, messages like tihs should be sent to a remote service (sentry?), but as long as we don't have anything like that a message in the console seems like the best possible alternative to me. 
To me, it's not only helpful in development, but also when users see the error in production and we're trying to analyze remotely.

Reply:
Ok. I can accept that :smiley:

Reply:
You could also use `warn()` of `@ember/debug` to hide it in production builds since we don't want to spam the users console, do we?